packages/opencensus-instrumentation-http/src/http-metrics.ts

```diff
@@ -0,0 +1,239 @@
+    'opencensus.io/http/client/received_bytes', HTTP_CLIENT_RECEIVED_BYTES,
+    AggregationType.DISTRIBUTION, [HTTP_CLIENT_METHOD, HTTP_CLIENT_STATUS],
+    'Size distribution of client-side HTTP response body', SIZE_DISTRIBUTION);
+
+/**
+ * {@link View} for roundtrip latency distribution of client-side HTTP requests.
+ */
+const HTTP_CLIENT_ROUNDTRIP_LATENCY_VIEW = globalStats.createView(
+    'opencensus.io/http/client/roundtrip_latency',
+    HTTP_CLIENT_ROUNDTRIP_LATENCY, AggregationType.DISTRIBUTION,
+    [HTTP_CLIENT_METHOD, HTTP_CLIENT_STATUS],
+    'Roundtrip latency distribution of client-side HTTP requests',
+    LATENCY_DISTRIBUTION);
+
+/** {@link View} for count of server-side HTTP requests serving completed. */
+const HTTP_SERVER_COMPLETED_COUNT_VIEW = globalStats.createView(
+    'opencensus.io/http/server/completed_count', HTTP_SERVER_LATENCY,
+    AggregationType.COUNT,
+    [HTTP_SERVER_METHOD, HTTP_SERVER_ROUTE, HTTP_SERVER_STATUS],
+    'Count of HTTP server-side requests serving completed');
+
+/** {@link View} for size distribution of server-side HTTP request body. */
+const HTTP_SERVER_RECEIVED_BYTES_VIEW = globalStats.createView(
+    'opencensus.io/http/server/received_bytes', HTTP_SERVER_RECEIVED_BYTES,
+    AggregationType.DISTRIBUTION,
+    [HTTP_SERVER_METHOD, HTTP_SERVER_ROUTE, HTTP_SERVER_STATUS],
+    'Size distribution of server-side HTTP request body', SIZE_DISTRIBUTION);
+
+/** {@link View} for size distribution of server-side HTTP response body. */
+const HTTP_SERVER_SENT_BYTES_VIEW = globalStats.createView(
+    'opencensus.io/http/server/sent_bytes', HTTP_SERVER_SENT_BYTES,
+    AggregationType.DISTRIBUTION,
+    [HTTP_SERVER_METHOD, HTTP_SERVER_ROUTE, HTTP_SERVER_STATUS],
+    'Size distribution of server-side HTTP response body', SIZE_DISTRIBUTION);
+
+/**
+ * {@link View} for latency distribution of server-side HTTP requests serving.
+ */
+const HTTP_SERVER_LATENCY_VIEW = globalStats.createView(
+    'opencensus.io/http/server/server_latency', HTTP_SERVER_LATENCY,
+    AggregationType.DISTRIBUTION,
+    [HTTP_SERVER_METHOD, HTTP_SERVER_ROUTE, HTTP_SERVER_STATUS],
+    'Latency distribution of server-side HTTP requests serving',
+    LATENCY_DISTRIBUTION);
+
+const HTTP_BASIC_SERVER_VIEWS: View[] = [
+  HTTP_SERVER_COMPLETED_COUNT_VIEW, HTTP_SERVER_RECEIVED_BYTES_VIEW,
+  HTTP_SERVER_SENT_BYTES_VIEW, HTTP_SERVER_LATENCY_VIEW
+];
+
+const HTTP_BASIC_CLIENT_VIEWS: View[] = [
+  HTTP_CLIENT_COMPLETED_COUNT_VIEW, HTTP_CLIENT_SENT_BYTES_VIEW,
+  HTTP_CLIENT_RECEIVED_BYTES_VIEW, HTTP_CLIENT_ROUNDTRIP_LATENCY_VIEW
+];
+
+/** Register all default views. */
+export function registerAllViews(globalStats: Stats) {
+  registerAllClientViews(globalStats);
+  registerAllServerViews(globalStats);
+}
+
+/** Register all default client views. */
+export function registerAllClientViews(globalStats: Stats) {
+  for (const CLIENT_VIEW of HTTP_BASIC_CLIENT_VIEWS) {
```

Comment:
Optional nit: should we use `ALL_CAPS` for `const` declarations that are local to a function (here and for `registerAllServerViews` below)?

Reply:
Done